src/content/docs/tutorial-create-alerts/manage-alert-quality.mdx

```diff
@@ -0,0 +1,182 @@
+
+<Callout variant="tip">
+Want to try a hands on learning approach before you start implementing this in your account? Check out the [alert quality management lab](https://learn.newrelic.com/hands-on-lab-alert-quality-management).
+</Callout>
+
+## Why use alert quality management? [#why-aqm]
+
+By using an alert strategy based on measuring business impact, you'll decrease response time and increase awareness of critical events. As you improve your alert signal to noise ratio, you'll reduce confusion and be able to rapidly identify and isolate the root cause of your problems. The overall goal of alert quality management is to lower the volume of less valuable alerts while providing easier ways to identify when more valuable incidents are created. This results in:
+
+* Increased uptime and availability.
+* Reduced mean time to resolution (MTTR).
+* Decreased alert volume.
+* The ability to easily identify alerts that are not valuable, so you can either make them valuable or remove them.
+
+AQM uses metrics to help you measure these, called **Key performance indicators**. For more on KPIs, see below.
+
+## Using Key performance indicators [#kpi]
+
+KPIs help you to find the noisiest and least valuable alerts so you can improve their value or remove them. You'll use the AQM process to collect and measure incident volume and engagement KPIs, and in turn, you'll use the long term metric trends to show real business impact to management and stakeholders. Below, you'll find information on all the KPIs, as well as a NRQL query for each one to help you monitor them from anywhere in the New Relic UI. 
+
+### Incident volume [#volume]
+
+You should treat incidents (with or without alerts) like a queue of tasks. Just like a queue, the number of alerts should always be as close to zero as possible. Each incident should trigger an investigatory or corrective action to resolve the condition. If an alert doesn't result in some sort of action, then you should question the value of the alert condition.
+
+In particular, if you see specific incidents that are frequently triggered, then you should question whether you're in a constant state of meaningful impact or if you simply have a large volume of noise? The incident volume KPIs help you answer those questions and measure progress towards a healthy state of high quality alerting.
+
+<CollapserGroup>
+  <Collapser
+    id="kpi-incident-count"
+    title="Incident count KPI"
+  >
+This is the number of incidents generated over a period of time. Typically you should compare the current and previous weeks.
+
+**Goal:** Reduce the number of low value / nuisance incidents.
+
+**Best practices:**
+* Ensure condition settings are intended to detect real business impact.
+* Ensure condition settings are detecting abnormal behavior.
+* Communicate that the incident details `Acknowledge` feature helps measure meaningful and actionable alerts. See [percentage incident acknowledge KPI](#kpi-user-engagement).
```

Comment:
`Communicate that the incident details `Acknowledge` feature helps measure meaningful and actionable alerts.`

The action the reader is meant to take here isn't really clear to me.